db/post_migrate/20200917193528_migrate_notifications_type.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class MigrateNotificationsType < ActiveRecord::Migration[5.2]
+  disable_ddl_transaction!
+
+  TYPES_TO_MIGRATE = %w(
+    Mention
+    Status
+    Status
```

Comment:
:thinking:

Reply:
Good catch, you also might have found another issue through this... `Notification::TYPE_CLASS_MAP` has `Status` two times on the right side, that means the `TYPE_CLASS_MAP.invert[activity_type]` starts looking unpredictable. Also, the logic of `activity_type.underscore` falls apart with the `reblog` type, which is one of the older ones, `status` type being the new addition.

Reply:
> Notification::TYPE_CLASS_MAP has Status two times on the right side, that means the TYPE_CLASS_MAP.invert[activity_type] starts looking unpredictable.

It is, in fact, predictable, and will take the second value, so it should be fine. Still, uncomfortable with it, as I said earlier, since it's not easily understood.